src/containers/inline-widget/index.js

```diff
@@ -0,0 +1,31 @@
+import { h } from "preact";
+import { CacheProvider } from "@emotion/react";
+
+import WidgetBase from "../base-class";
+import Title from "../../components/title";
+import { StyledCache } from "../../utils/styled-cache";
+import { WidgetWrapper } from "../../utils/widget-wrapper";
+import { DEFAULT_BASE_CLASSNAME } from "../../utils/constants";
+
+class InlineWidget extends WidgetBase {
+  render({}, { styles, title, logo }) {
+    return (
+      <WidgetWrapper
+        customStyles={styles}
+        className={`${DEFAULT_BASE_CLASSNAME}-wrapper`}
+      >
+        <CacheProvider
+          value={StyledCache("inline-fourohtwo", `.inline-fourohtwo`)}
```

Comment:
can you explain again why we need different classes here?

Reply:
where do we have potentially conflicting styles?

Reply:
The two different classnames are required since we are scoping the styles of each widget based on the selector which we provide to the widget renderer.

If we go with the same classnames, then the 2nd widget isn't getting any styles (tested locally and refer to screenshot below).

So, to avoid that issue, it is better to scope the styles differently for both the widgets.

![image](https://user-images.githubusercontent.com/3415488/104815811-c30b0d80-580e-11eb-9bcc-bb1e32e0ba6d.png)